recipes/libcufft/meta.yaml

```diff
@@ -0,0 +1,137 @@
+{% set name = "libcufft-split" %}
+{% set version = "11.0.0.21" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/libcufft/{{ platform }}/libcufft-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 4650f79fcf377af410afedb56445f1332e8f213ec3c94d660f9a91328c3ecb4a  # [linux64]
+  sha256: e711671995e9d92d16015a4cd3751ec30710a49fa22f2b88a4a42905c248d861  # [ppc64le]
+  sha256: 91aa1ba5e68ca65778222c9a427a9214c4550d2f45e978e51779a207bafc17a3  # [aarch64]
+  sha256: a33f577d48ef432d5cfa8d0ebfa789cc5261af5dd84fd9c608235edc3f769e43  # [win]
+
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+
+script: install.sh   # [linux]
+script: install.bat    # [win]
+
+
+outputs:
+  - name: libcufft
+    files:
+      - lib/libcufft*.so.*    # [linux]
+      - targets/*/lib/*.so.*  # [linux]
+      - Library\bin           # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+      host:                                    # [linux]
+        - sysroot_{{ target_platform }} 2.17   # [linux]
+        - __glibc >=2.17                       # [linux]
+      run:                                     # [linux]
+        - sysroot_{{ target_platform }} 2.17   # [linux]
+        - __glibc >=2.17                       # [linux]
```

Comment:
```suggestion
      host:
        - sysroot_{{ target_platform }} 2.17   # [linux]
        - __glibc >=2.17                       # [linux]
        - cuda-version {{ cuda_version }}
      run:
        - sysroot_{{ target_platform }} 2.17   # [linux]
        - __glibc >=2.17                       # [linux]
        - {{ pin_compatible("cuda-version") }}
```

Reply:
Would tweak this slightly to move `sysroot` into `requirements/build` (with the compilers) and drop `glibc` & `sysroot` from other sections. Typically `sysroot` is added next to the compilers (this is different for compiler packages like `cuda-nvcc`). Also `glibc` would be added by `sysroot` automatically so shouldn't need hard-coding after this change

Reply:
Thanks, will use this as is and then compare with cublas later today.

Reply:
Thanks @jakirkham, I wasn't able to concisely edit all the pieces at once. I agree with your suggestions, just wanted to get `cuda-version` fixed first.

Reply:
I added a `, max_pin="x.x")` to one of these.  Confirmed that the contents look fine.  The setup warns that the `cxx` requirement may not be needed.

Not sure about:
1. Duplication of about section (I thought it gets inherited, but sounds like its not perfectly)?
2. cudart dependency for the dev versions (we need the cudart headers), either choice makes sense.